index.html

```diff
@@ -49,198 +62,199 @@ <h1>I'm Alier Glad to see you.</h1>
-                <li class="item1">
-                  <img src="images/Frame9.png" alt="image" />
-                </li>
-                <li class="item2">
-                  <img src="images/Frame10.png" alt="image" />
-                </li>
-                <li class="item3">
-                  <img src="images/Frame11.png" alt="image" />
-                </li>
-              </ul>
-            </div>
-            <div class="frame">
-              <h3>Frameworks</h3>
-              <img src="images/union1.png" alt="icon" />
-            </div>
-            <div class="skill">
-              <h3>Skills</h3>
-              <img src="images/union1.png" alt="icon" />
-            </div>
+      <!--About-me-section begins here-->
+
+      <section class="about-me">
+        <div class="about-text-content-col">
+          <h1 class="about-text">
+            About <br />
+            Myself
+          </h1>
+
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+
+          <p class="lc">LET'S CONNECT</p>
+
+          <ul class="social-icon">
+            <li>
+              <a href=""><img src="images/vector.svg" alt="image" /></a>
+            </li>
+            <li>
+              <a href=""><img src="images/in.svg" alt="image" /></a>
+            </li>
+            <li>
+              <a href=""><img src="images/vectorhand.png" alt="image" /></a>
+            </li>
+            <li>
+              <a href=""><img src="images/vectorhead.png" alt="image" /></a>
+            </li>
+            <li>
+              <a href=""><img src="images/vectorm.png" alt="image" /></a>
+            </li>
+          </ul>
+
+          <div class="resume">
+            <p><a href="">Get my resume</a></p>
+          </div>
+        </div>
+        <div class="lang-col">
+          <div class="lang">
+            <h3>Languages</h3>
+            <img src="images/Union.png" alt="icon" />
+          </div>
+
+          <ul class="tech">
+            <li>
+              <img src="images/javascript.png" alt="image" />
+            </li>
+            <li>
+              <img src="images/html-logo.png" alt="image" />
+            </li>
+            <li>
+              <img src="images/cssLogo.png" alt="image" />
+            </li>
+          </ul>
+
+          <div class="frame">
+            <h3>Frameworks</h3>
+            <img src="images/union1.png" alt="icon" />
+          </div>
+          <div class="skill">
+            <h3>Skills</h3>
+            <img src="images/union1.png" alt="icon" />
           </div>
         </div>
       </section>
-    </main>
-    <!--Contact form begins here-->
-    <div class="background-container">
+
+      <!--Contact form begins here-->
+
       <section class="contact-me--form">
```

Comment:
- [ ] I think here for the contact section you need to align the elements to be centered as the Figma design so according to the requirements please stick with the Figma as much as possible and Fix the alignment here you can use `flexbox` for that .

- [ ] I noticed here the inputs don't have the same width as the Figma kindly Fix the width,  and I think it would be better if you decrease the `border-radius` to the inputs to match the Figma 🙏🏻🙏🏻🔥🔥

- [ ] I think the button also need more width to match the design and to make text-align to the center so please fix it as well 🔥🔥🙏🏻🙏🏻

- Your design
![image](https://user-images.githubusercontent.com/58553711/178123542-7224265f-f624-4b79-a1e9-9ab481dbb219.png)

- Figma design 

![formff](https://user-images.githubusercontent.com/58553711/178123528-621d1128-9c6d-47cb-be14-f7cab908665e.PNG)